frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/backgroundables/SlideInBackgroundable.java

```diff
@@ -136,13 +142,11 @@ private void waitForCreationOfLocalBranch() {
     }
   }
 
-  private Consumer<Throwable> exceptionWithMessageHandler(@Nullable String message) {
-    return t -> {
-      notifier.notifyError(
-          /* title */ format(getString("action.GitMachete.BaseSlideInBranchBelowAction.notification.title.slide-in-fail"),
-              slideInOptions.getName()),
-          message != null ? message : getMessageOrEmpty(t));
-    };
+  private void exceptionWithMessageHandler(@Nullable String message, Throwable throwable) {
```

Comment:
Okay... and now I'd also rename this method adequately (`notifyError`?)

Reply:
:facepalm: ofc, sorry